docs/cmd/tkn_pipeline_logs.md

```diff
@@ -15,17 +15,17 @@ Show pipeline logs
 ### Examples
 
 
-  # show logs interactively for no inputs
+  # interactive mode: shows logs of the selected pipeline run
```

Comment:
Won't this be rendered as H1? 

# interactive mode: ...

Reply:
yes, need to change example in the respective file

Reply:
Yeah, those need to be updated in the source code (maybe we can remove some and add those in a md file so it only appears on man pages and reference docs)